standards/distributions.Rmd

```diff
@@ -51,6 +51,8 @@ heading](https://cran.r-project.org/web/views/Distributions.html#Misc) on
 *Unified interface to handle distributions*. Packages mentioned in that
 sub-section include:
 
+- The core [`stats`](https://cran.r-project.org/web/packages/STAT/index.html) 
+  package built into base `R`;
```

Comment:
```suggestion
  package distributed wtih base `R`;
```
Thanks Grant, just a minor suggestion because "base R" is used to refer to just that, while "stats" and others are referred to as "recommended packages" distinct from "base R" itself.